app/controllers/admin/categories_controller.rb

```diff
@@ -59,13 +59,16 @@ def category_params
     :front_page,
     :active,
     :section,
-    :rank
+    :rank,
+    :team_list,
```

Comment:
@scott, I just added this featured to create a filter by team, but looking on feature, I think, we need to make a review on this feature on Internal KB.